tests/appsec/test_traces.py

```diff
@@ -53,8 +53,8 @@ def test_custom_span_tags(self):
         """AppSec should store in APM spans some tags when enabled."""
 
         def validate_custom_span_tags(span):
-            if span.get("name") != "init.service":
-                return
+            if span.get("name") not in ("init.service", "web.request"):
+                return True
```

Comment:
Returning true here means that it will validate the test if it founds any span with name not in  ("init.service", "web.request"), which is not what we want.

Return None means that the validation is not validated neither invalidated.

Furthermore, I thnink alsmost all span as name = `web.request`, so it's probably not what we want. Can you simply skip the test for now, I'll have a look.